### PR TITLE
Update main with changes through v0.76.3

### PR DESCRIPTION
**Pull request by djaglowski**
## **IMPORTANT**: The pull request to bring the changes from the release branch *MUST* be merged using the **Merge pull request** method, and *NOT* squashed using “**Squash and merge**”. This is important as it allows us to ensure the commit SHA from the release branch is also on the main branch. **Not following this step will cause much go dependency sadness.**

### CODE REVIEW

**Review by atoulme — APPROVED** [review]

File: cmd/oteltestbedcol/builder-config.yaml [hunk 1]
```diff
@@ -2,50 +2,50 @@ dist:
   module: github.com/open-telemetry/opentelemetry-collector-contrib/cmd/oteltestbedcol
   name: oteltestbedcol
   description: OpenTelemetry Collector binary for testbed only tests.
-  version: 0.76.1-dev
+  version: 0.76.2-dev
```
Comment (TylerHelmuth) [inline cmd/oteltestbedcol/builder-config.yaml:5]: Does this need to be `0.76.3-dev`?

File: cmd/oteltestbedcol/main.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ func main() {
 	info := component.BuildInfo{
 		Command:     "oteltestbedcol",
 		Description: "OpenTelemetry Collector binary for testbed only tests.",
-		Version:     "0.76.1-dev",
```
Comment (MovieStoreGuy) [inline cmd/oteltestbedcol/main.go:22]: This is using the wrong version?

File: internal/tools/go.sum [hunk 3]
```diff
@@ -750,8 +750,8 @@ go.opentelemetry.io/build-tools/issuegenerator v0.7.0 h1:p4V6PhTQmp5kWBRqYf+bd6k
 go.opentelemetry.io/build-tools/issuegenerator v0.7.0/go.mod h1:dBqO2CqMnVm35etsxOYZqNtA+4ds5wGoELSAsau7itY=
 go.opentelemetry.io/build-tools/multimod v0.7.0 h1:Af1wVNaJRMVqRkiiRjeMrAfKJDejvpsLoBF9H5UrjSo=
 go.opentelemetry.io/build-tools/multimod v0.7.0/go.mod h1:WAwBtJC42JbRDzfi3erCYQ9M1RnyoIcqI/H8eqBBPQo=
-go.opentelemetry.io/collector/cmd/builder v0.75.0 h1:oAMeqGm4IfNeFKNXspmVCp9qK6CvBe1Ag+0hx4cAIZA=
-go.opentelemetry.io/collector/cmd/builder v0.75.0/go.mod h1:tIITwEHJUrMfMe13obMA9C81CqR4eiXB3unf5rZdXxE=
+go.opentelemetry.io/collector/cmd/builder v0.76.1 h1:CbKqIud0XAZuMcj4xyB5GDJkJ4xQsbN1yclox+wPIbM=
```
Comment (MovieStoreGuy) [inline internal/tools/go.sum:753]: Something is trying to pull in an older version of the builder?

**Review by MovieStoreGuy — COMMENTED** [review]
Just a couple things.

File: internal/tools/go.sum (hunk 3 above)
Comment (TylerHelmuth) [reply internal/tools/go.sum:753]: The builder lives in core and is 0.76.1.

Comment (MovieStoreGuy) [reply internal/tools/go.sum:753]: Didn't catch that, my visual grep was for anything not `v0.76.3`

**Review by codeboten — COMMENTED** [review]
Superseeded by https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21287

**Pull request closed without merging**